app/controllers/home_controller.rb

```diff
@@ -1,4 +1,13 @@
 class HomeController < ApplicationController
   def index
+    if current_user
+      if current_user.is_admin
+        redirect_to questions_path
+      else
+        redirect_to my_profile_path
+      end
+    else
+      redirect_to new_user_session_path
+    end
```

Comment:
current_user.is_admin must be called after testing if current_user exists (is not nil)